### DISCUSSION TITLE
Missing "string" when rendering livewire.js makes error Unexpected Identifier Happen

### DISCUSSION THREAD

**Discussion by KhairulAzmi21** [livewire/livewire#3532] [7. Bugs (archived)] [answered]
Livewire Version 2.4 . 

![Screenshot 2021-08-17 at 10 31 13 AM](https://user-images.githubusercontent.com/12378983/129654571-3d6f92af-7221-4498-abf3-9bcf63cfe214.png)

I got this error when I run on the staging site . I was working perfectly locally . 

happen to be, I found out the is some missing string on the livewire.js
![Screenshot 2021-08-17 at 10 29 08 AM](https://user-images.githubusercontent.com/12378983/129654640-c45930d1-b483-4505-bd4c-88b665cd92a7.png)

the "n" was missing on the "string"

`string){return String(string).replace(replacement,".").toLowerCase()},data=isForced.data=` . I just copy paste this .

**Comment by ryatkins** [comment, marked as answer]
Can you test on Livewire 2.5.5 ?

**Reply by joshhanley** [reply]
@KhairulAzmi21 yeah there was some bugs in previous versions. Please test on 2.5.5 as suggested by Ryan.
